src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java

```diff
@@ -23,6 +27,9 @@ public class PiattaformaNotifichePage extends BasePage {
     private static final Logger logger = LoggerFactory.getLogger("notificaMittentePagoPA");
     private final List<NetWorkInfo> netWorkInfos = Hooks.netWorkInfos;
 
+    private final NotificationSingleton notificationSingleton = new NotificationSingleton();
```

Comment:
This is not how a singleton works.

Please take a look at this suggestion and see if the singletong method is neccessary otherwise you can just simple remove it:
```suggestion
    private final NotificationSingleton notificationSingleton = NotificationSingleton.getInstance();
```

Reply:
done